docs/topics/class-based-views/generic-editing.txt

```diff
@@ -88,10 +88,11 @@ If you want to use a custom :class:`~django.forms.ModelForm` (for instance to
 add extra validation) simply set
 :attr:`~django.views.generic.edit.FormMixin.form_class` on your view.
 
-.. note::
-    When specifying a custom form class, you must still specify the model,
-    even though the :attr:`~django.views.generic.edit.FormMixin.form_class` may
-    be a :class:`~django.forms.ModelForm`.
+.. versionchanged:: 1.10
+
+    When specifying a :class:`~django.forms.ModelForm` as
```

Comment:
In older versions, you need to specify...
(also add the new logic to the bulleted list above as well as a mention in the 1.10 release notes)